cli/types/cypress.d.ts

```diff
@@ -3466,13 +3466,15 @@ declare namespace Cypress {
   type ConfigHandler<T> = T
   | (() => T | Promise<T>)
 
+  type CustomFramework = `cypress-ct-${string}` | `@${string}/cypress-ct-${string}`
```

Comment:
Since we now use eslint (same as rest of repo) instead of the deprecated `tslint`, the types now match the rest of the formatting rules in our repo.